.nvmrc

```diff
@@ -1 +1 @@
-14
+^16
```

Comment:
what's this for? I'd revert as we don't wanna force contributors into node v16

Reply:
Sorry, I made a mess - this was a failed attempt at convincing Heroku not to install latest Node when building the app.
After I failed at that, I removed that repo so can't continue with this PR.
Please feel free to sneak the code into some other PR or commit.